community/dags/dag_stock_comments.py

```
@@ -0,0 +1,90 @@
+# src 디렉토리를 Python 경로에 추가
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', 'src'))
+
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.utils.task_group import TaskGroup
+from airflow.operators.dummy import DummyOperator
+from crawler import MultiStockYahooFinanceCrawler
+from config import stocks, logs_base_dir
+import psycopg2
+from psycopg2.extras import execute_values
```

Ruff (0.12.2)

`psycopg2.extras.execute_values` imported but unused

Remove unused import: `psycopg2.extras.execute_values`

(F401)